test/org/sagebionetworks/bridge/services/SharedModuleServiceTest.java

```diff
@@ -99,6 +102,7 @@ public void byIdAndVersionSchemaSuccess() {
         assertEquals(SharedModuleType.SCHEMA, status.getModuleType());
         assertEquals(SCHEMA_ID, status.getSchemaId());
         assertEquals(SCHEMA_REV, status.getSchemaRevision().intValue());
+        assertTrue(sharedSchema.getPublished());
```

Comment:
This is not how we want to test this either. This tests that the schema returned by the getUploadSchema call has its published value set to true. This tests the implementation, not the behavior. We want to test that the schema passed into the createSchema() call has its published value set to true. (Here and in the other place.)